### PR TITLE
Label flaky tests and add early returns in setTimeout functions to bypass undefined fullLayouts

### PR DESCRIPTION
**Pull request by archmoj**
The goal of this PR is improving the success rate of `test-jasmine` container mainly by tagging `flaky` test so that they run `test-jasmine3` container and reduce overall failures.

@plotly/plotly_js

### CODE REVIEW

File: src/plots/cartesian/index.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ exports.finalizeSubplots = function(layoutIn, layoutOut) {
  */
 exports.plot = function(gd, traces, transitionOpts, makeOnCompleteCallback) {
     var fullLayout = gd._fullLayout;
+    if(!fullLayout._subplots) return;
```
Comment (alexcjohnson) [inline src/plots/cartesian/index.js:129]: What's this about? When do we encounter this?

Comment (archmoj) [reply src/plots/cartesian/index.js:129]: Sometimes happen at the end of `transition_test` with an error similar to the image below:
undefinedSubplot

Comment (archmoj) [reply src/plots/cartesian/index.js:129]: Simply for reference:
https://app.circleci.com/pipelines/github/plotly/plotly.js/2974/workflows/734c1977-517c-43b8-88bf-9fe83e3c5e55/jobs/103632

**Review by alexcjohnson — APPROVED** [review]
Very nice! 💃

**Pull request merged**